enclave_manager/Makefile

```diff
@@ -19,15 +19,24 @@ ENCLAVE_TYPE?=singleton
 PY_VERSION=${shell python3 --version | sed 's/Python \(3\.[0-9]\).*/\1/' | tr -d .}
 MOD_VERSION=${shell ../bin/get_version}
 
-WHEEL_FILE=dist/${ENCLAVE_TYPE}_enclave_manager-${MOD_VERSION}-cp${PY_VERSION}-cp${PY_VERSION}m-linux_x86_64.whl
+ENCLAVE_MANAGER_WHEEL_FILE=dist/${ENCLAVE_TYPE}_enclave_manager-${MOD_VERSION}-cp${PY_VERSION}-cp${PY_VERSION}m-linux_x86_64.whl
+ENCLAVE_COMMON_WHEEL_FILE=dist/enclave_manager_common-${MOD_VERSION}-cp${PY_VERSION}-cp${PY_VERSION}m-linux_x86_64.whl
```

Comment:
Should we keep a separate Makefile for common. Else if we build KME and WPE for  the same common wheel file might get build twice. Just cross check. May be Makefile build system knows there is no change to teh common wheel file and might not build it twice.

Reply:
it has just one swig file so thought of  include it in the same Makefile. I will consider this splitting to separate Makefile in the next PR, because i will create one more swig file for dcap.